content/blog/2021/09/2021-09-21-jenkins-at-devops-world.adoc

```diff
@@ -0,0 +1,78 @@
+---
+:layout: post
+:title: "Join Jenkins at DevOps World 2021"
+tags:
+- events
+- conference
+- cicd
+- talks
+- workshops
+author: alyssat
+description: >
+  DevOps World 2021 will include significant Jenkins workshops, sessions, and interactions.
+  Join us September 28-30, 2021 for Jenkins workshops, presentations, and conversations.
+opengraph:
+  image: /images/conferences/devops-world-2021.png
+links:
+  discourse: true
+---
+
+link:https://www.devopsworld.com/[DevOps World] has been the largest gathering for Jenkins for many years.
```

Comment:
```suggestion
link:https://www.devopsworld.com/[DevOps World] has been the largest gathering for Jenkins for many years.
Because of Covid-19 and to allow more people to attend, DevOps World will again be a virtual event.
```